.github/workflows/coprocessor-benchmark-gpu.yml

```diff
@@ -232,8 +232,9 @@ jobs:
           ls
           pwd
           cp ./host-contracts/.env.example ./host-contracts/.env
-          npm --prefix ./host-contracts ci --include=optional
-          cd host-contracts && npm install && npm run deploy:emptyProxies && npx hardhat compile
+          cd ./host-contracts 
+          npm ci --include=optional
+          npm install && npm run deploy:emptyProxies && npx hardhat compile
```

Comment:
There was still a failure in the bench last time I checked, is this really enough?

Reply:
the last failure was a compilation error due to the params struct that wasn't updated in coprocessor (I pushed updates to fhevm for that)